README.md

```diff
@@ -10,8 +10,6 @@
 
 - **Applicable**: Dashboards
```

Comment:
Can you remove this dashboard line, not applicable on dashboard

Reply:
ok. Fixed.